buildSrc/src/main/kotlin/com.hedera.block.jpms-modules.gradle.kts

```diff
@@ -167,6 +167,14 @@ extraJavaModuleInfo {
     module("org.jetbrains.kotlinx:kotlinx-metadata-jvm", "kotlinx.metadata.jvm")
 
     // Test clients only
+    module("com.github.docker-java:docker-java-api", "com.github.dockerjava.api")
+    module("com.github.docker-java:docker-java-transport", "com.github.dockerjava.transport")
+    module(
+        "com.github.docker-java:docker-java-transport-zerodep",
+        "com.github.dockerjava.transport.zerodep"
+    )
+    module("org.slf4j:slf4j-api", "org.slf4j") { patchRealModule() }
```

Comment:
Not 100% sure about this, but it silenced the error and doesn't brake anything else.